instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/utils/location/MockLocationReplayer.kt

```diff
@@ -52,6 +53,20 @@ class MockLocationReplayerRule(mockLocationUpdatesRule: MockLocationUpdatesRule)
         mapboxReplayer?.seekTo(replayEvents.first())
         mapboxReplayer?.play()
     }
+
+    fun loopUpdate(location: Location, times: Int) {
+        val events: List<ReplayEventUpdateLocation> =
+            mutableListOf<ReplayEventUpdateLocation>().apply {
+                for (i in 0..times) {
+                    this.add(location.toReplayEventUpdateLocation(i.toDouble()))
+                }
```

Comment:
```suggestion
                repeat(times) {
                    this.add(location.toReplayEventUpdateLocation(i.toDouble()))
                }
```

Reply:
Ah! I knew Kotlin definitely had something like this :+1: